### ISSUE TITLE
Usage without Docker?

### ISSUE THREAD

**Issue by Levaru** [ymd-stella/godot_ros_pcviz#2]
I've never used Docker before and noticed that it basically downloads Ros again and creates its own workspace which is not desirable in my case.

I did manage to bypass it by generating the godot-cpp bindings manually and then successfully build the package with catkin.

I then imported it into Godot and tried to run it but got an error that libxmlrpc.so could not be found. How should my setup look like to get this working?

**Comment by ymd-stella** [comment]
> I did manage to bypass it by generating the godot-cpp bindings manually and then successfully build the package with catkin.

Please give me more detailed information to reproduce in my environment.

**Comment by Levaru** [comment]
I have a standard catkin workspace with multiple ROS packages. 
I cloned your godot_ros_pcviz package, and updated the git submodules.

Then i compiled the godot-cpp bindings library with the following command:
`scons platform=linux bits=64 generate_bindings=yes`

Next i had to remove `tf2_sensor_msgs` from CMakeLists.txt and package.xml since that package doesn't exist under ROS-Melodic.

After this i successfully build my workspace with `catkin build`. I then downloaded the newest Godot executable, ran it and imported the `project.godot` file.

When executing the project i get the following errors:
```
Can't open dynamic library: /home/user/catkin_ws/devel/lib/godot_ros_pcviz/libgodot_ros_pcviz.so. Error: libxmlrpcpp.so: cannot open shared object file: No such file or directory
No valid library handle, can't get symbol from GDNative object
No nativescript_init in "res://libgodot_ros_pcviz.so" found
```

Looks like im missing something here but looking at your `entrypoint.sh` i basically did the same thing just without a docker volume.
